CMakeLists.txt

```diff
@@ -347,6 +347,14 @@ if(BUILD_LIBRARY)
     endif()
 endif()
 
+INCLUDE(FetchContent)
+FetchContent_Declare(
+    fmt
+    GIT_REPOSITORY https://github.com/fmtlib/fmt.git
+    GIT_TAG master
```

Comment:
This is not a stable ref.

Is it possible to vendor a specific version like we do in `depends/` for some other libs?

Reply:
We can select a specific tag, yes - 12.1.0 is the most recent release and if we selected that tag (instead of "master") it would use that as a stable pin